src/index.ts

```diff
@@ -126,10 +126,10 @@ class Index {
 
     async getManifestsFromS3(bucketName: string, prefix: string): Promise<Manifest[]> {
         const s3 = new S3({apiVersion: '2006-03-01'})
-        const result = await util.promisify(s3.makeUnauthenticatedRequest.bind(s3))('listObjectsV2', {
+        const result: S3.Types.ListObjectsV2Output = await util.promisify(s3.makeUnauthenticatedRequest.bind(s3, 'listObjectsV2', {
```

Comment:
will this logic have to change once we move away from AIM users to specifying auth in the bucket policy (as discussed with Rajan) ?

Reply:
Possibly. I should probably be looped in on the bucket policy. Right now, ListObjects is special-cased as being allowed to work without authentication, which was just the fastest way of making this work back in the day

Reply:
Got it - for now, there is nothing new to loop you in on. As soon as I have more info tho, I will update you. We still have yet to write the tech spec for the infrastructure rehaul